enatega-multivendor-app/src/components/Main/TopBrands/styles.js

```diff
@@ -6,38 +6,51 @@ const { height } = Dimensions.get('window')
 
 const styles = (props = null) =>
   StyleSheet.create({
-    topbrandsSec:{
-        ...alignment.PLmedium,
-    },
-    brandImg:{
-        width: '100%', 
-        // aspectRatio: 18/8,
-        height: scale(70),
-        objectFit: 'cover'
-      },
-      topbrandsContainer:{
-        flexGrow: 1,
-        width: scale(90), 
-        marginTop:scale(7),
-        ...alignment.MRmedium      
-      },
-      brandImgContainer:{
-        backgroundColor: '#F3F4F6',
-        borderWidth:1,
-        borderColor:'#E5E7EB',
-        borderRadius:8,
-        // padding:scale(8),
-      },
-      brandName:{
-        marginTop:scale(5),
-        marginBottom:scale(2)
-      },
-      margin: {
-        ...alignment.MLmedium,
-        ...alignment.MBmedium
-      },
-      
-      
+    topbrandsSec: {
+      ...alignment.PLmedium
+    },
+    brandImg: {
+      width: '100%',
+      // aspectRatio: 18/8,
+      height: scale(70),
+      objectFit: 'cover'
+    },
+    topbrandsContainer: {
+      flexGrow: 1,
+      width: scale(90),
+      marginTop: scale(7),
+      ...alignment.MRmedium
+    },
+    brandImgContainer: {
+      backgroundColor: '#F3F4F6',
+      borderWidth: 1,
+      borderColor: '#E5E7EB',
+      borderRadius: 8
+      // padding:scale(8),
+    },
+    brandName: {
+      marginTop: scale(5),
+      marginBottom: scale(2)
+    },
+    margin: {
+      ...alignment.MLmedium,
+      ...alignment.MBmedium
+    },
+    screenBackground: {
+      backgroundColor: props != null ? props.themeBackground : '#FFF',
+      ...alignment.PBlarge
+    },
+    placeHolderFadeColor: {
```

Comment:
By making separate component as mentioned above, get rid of all styles for loading screen in all component files.